src/note-context.jsx

```diff
@@ -0,0 +1,67 @@
+import { createContext, useContext } from "react";
+import { useReducer } from "react";
+
+const NoteContext = createContext();
```

Comment:
Suggestion- you can create a single data-context for all functionalities like notes,acrhives, labels etc.
No need of creating separate context for each.

Reply:
thanks